modules/EnsEMBL/Web/SpeciesDefs.pm

```diff
@@ -631,6 +632,7 @@ sub _expand_database_templates {
   ## Autoconfigure databases
   unless (exists $tree->{'databases'} && exists $tree->{'databases'}{'DATABASE_CORE'}) {
     my @db_types = qw(CORE CDNA OTHERFEATURES RNASEQ FUNCGEN VARIATION);
+    push @db_types, 'COMPARA' if $SiteDefs::SINGLE_SPECIES_COMPARA;
```

Comment:
just checking, should the db_types be compara or pan_compara?

Reply:
It should be compara - each species has a separate compara database, so it has to be processed with the other species-specific databases. There's also a separate "pan compara" database that contains reference species from www and the NV divisions.